builder/frameworks/arduino.py

```
@@ -28,75 +28,119 @@
 import os
 import sys
 import shutil
-from os.path import join, exists
+import hashlib
+from os.path import join, exists, isabs, splitdrive, commonpath, relpath
 
 from SCons.Script import COMMAND_LINE_TARGETS, DefaultEnvironment, SConscript
```

Ruff (0.11.9)

`SCons.Script.COMMAND_LINE_TARGETS` imported but unused

Remove unused import: `SCons.Script.COMMAND_LINE_TARGETS`

(F401)